draft-ietf-core-cf-reg-update.md

```diff
@@ -197,6 +197,12 @@ In all other cases, the policy is "Expert Review," following the procedure descr
 
 A new column with the title "Notes" has been added to the CoAP Content-Formats Registration Procedures shown in {{tbl-new-cf-proc}}.
 
+The following note is added to the registry:
+
+> "Note: As per {{&SELF}}, Designated Experts can approve temporary registrations in the Expert Review range that are not subject to the formal {{-iana-early}} renewal process."
```

Comment:
Maybe change to singular "a Designated Expert can approve ..." ?
This makes it clear that a single expert can do so. Singular form is also used in the other changes below.

Reply:
I think using the plural here is more correct.
It doesn’t designate a quorum of DEs, so there’s no risk of misinterpreting it for a requirement in such sense.
It just says that someone with the DE role has the power to approve temp registrations that are exempt from 7120-defined process.
Changing it to the singular would read like we allow a DE to exceptionally do that, which is not what we mean.


Maybe:

> _”As per {{&SELF}}, temporary registrations in the Expert Review range are approved by Designated Experts. These registrations are not subject to the formal {{-iana-early}} renewal process."_

Reply:
I moved the note to the IANA considerations ([7de1743](https://github.com/core-wg/cf-reg-update/pull/56/commits/7de1743d10028a2ff6155846cdcf318d350ef727)), which is seems more fit.